cadc-registry-server/src/main/java/org/opencadc/reg/server/ServiceAvailability.java

```diff
@@ -85,6 +84,11 @@ public void setAppName(String string) {
         //no-op
     }
 
+    @Override
+    public void heartBeat() {
```

Comment:
I'd prefer to see it spelt heartbeat  (no capital B)

Reply:
Changed to 'heartbeat'.